lib/fog/vsphere/models/compute/server.rb

```diff
@@ -17,46 +17,47 @@ class Server < Fog::Compute::Server
         # In particular:
         #   GuestInfo: information about the guest operating system
         #   VirtualMachineConfigInfo: Access to the VMX file and configuration
-
-        attribute :name
-        # UUID may be the same from VM to VM if the user does not select (I copied it)
-        attribute :uuid
-        attribute :hostname
-        attribute :operatingsystem
-        attribute :public_ip_address, :aliases => 'ipaddress'
-        attribute :power_state,   :aliases => 'power'
-        attribute :tools_state,   :aliases => 'tools'
-        attribute :tools_version
-        attribute :mac_addresses, :aliases => 'macs'
-        attribute :hypervisor,    :aliases => 'host'
+        attribute :cluster
         attribute :connection_state
-        attribute :mo_ref
-        attribute :path
-        attribute :memory_mb
-        attribute :cpus
         attribute :corespersocket
-        attribute :interfaces
-        attribute :volumes
+        attribute :cpuHotAddEnabled
+        attribute :cpus
         attribute :customvalues
-        attribute :overall_status, :aliases => 'status'
-        attribute :cluster
         attribute :datacenter
-        attribute :resource_pool
-        attribute :instance_uuid # move this --> id
+        attribute :firmware
         attribute :guest_id
         attribute :hardware_version
-        attribute :scsi_controller # this is the first scsi controller. Right now no more of them can be used.
-        attribute :cpuHotAddEnabled
+        attribute :hostname
+        attribute :hypervisor,    :aliases => 'host'
+        attribute :instance_uuid # move this --> id
+        attribute :interfaces
+        attribute :mac_addresses, :aliases => 'macs'
+        attribute :memory_mb
         attribute :memoryHotAddEnabled
-        attribute :firmware
+        attribute :mo_ref
+        attribute :name
+        attribute :operatingsystem
+        attribute :overall_status, :aliases => 'status'
+        attribute :path
+        attribute :power_state,   :aliases => 'power'
+        attribute :public_ip_address, :aliases => 'ipaddress'
+        attribute :resource_pool
+        attribute :scsi_controller # this is the first scsi controller. Right now no more of them can be used.
+        attribute :summary # VirtualMachineSummary
+        attribute :tools_state,   :aliases => 'tools'
+        attribute :tools_version
+        # UUID may be the same from VM to VM if the user does not select (I copied it)
+        attribute :uuid
+        attribute :volumes
```

Comment:
Yes. I use a plugin for RubyMine (I recommend it) called, surprisingly, sort lines that makes it very easy. :)